tyk-pro/templates/_helpers.tpl

```diff
@@ -41,7 +41,7 @@ http
 
 {{- define "tyk-pro.dash_proto" -}}
 {{- if .Values.gateway.tls -}}
-https
+http
```

Comment:
This looks weird. Could you please elaborate?

Reply:
Yeah this is a flat out mistake.  in the above logic, if you set `gateway.tls=true` then it sets the dashboard url to `https` but NOT the dashboard itself, because that is controled by `dashboard.tls`

Reply:
We'll merge this for now, but the entire `tyk-pro.dash_proto` helper is then pointless and should be removed I think.